src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,13 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService userService = new UserService();
```

Comment:
services should be `private final` class-level fields
```suggestion
     */
    private final UserService userService = new UserService();
    
    public boolean login(String email, String password) {
```